go/src/koding/kites/kloud/stack/provider/describe_test.go

```diff
@@ -3,18 +3,21 @@ package provider_test
 import (
 	"reflect"
 	"testing"
+	"time"
 
 	"koding/kites/kloud/provider/aws"
 	"koding/kites/kloud/stack"
 	"koding/kites/kloud/stack/provider"
 )
 
 type Foo struct {
-	File []byte `kloud:",secret"`
-	N    int    `kloud:",readOnly"`
-	S    string
-	M    map[string]interface{} `kloud:"someMap"`
-	E    FooEnum                `json:"enums" kloud:"someEnums"`
+	File          []byte `kloud:",secret"`
+	N             int    `kloud:",readOnly"`
```

Comment:
Nope, just one-letter names due to laziness - `n` is usually int, `s` is usually string etc.